lib/wpc/include/tsap.h

```diff
@@ -0,0 +1,232 @@
+/* Wirepas Oy licensed under Apache License, Version 2.0
+ *
+ * See file LICENSE for full license details.
+ *
+ */
+
+#ifndef TSAP_H_
+#define TSAP_H_
+
+#include <stdint.h>
+
+/**
+ *  \brief Reservation for maximum test data size in radio interface
+ *
+ *  Test data header includes length (1 Byte) and sequence
+ *  number (4 Bytes) which allocates fixed 5 bytes from over the air
+ *  payload, remaining bytes can be used for free form data.
+ *
+ *  Maximum data size depends on used radio hardware. Here we allocate
+ *  big enough buffer to cover all supported radio hardwares and
+ *  some extra bytes for future needs.
+ *
+ *  True size of the radio payload can be read with ????test_lib->getMaxDataSize().
```

Comment:
What is the prpose of ??? in comment?

Reply:
Library name replaced with tsap_getRadioMaxDataSize() in next commit.